trilium_alchemy/core/entity/model.py

```diff
@@ -171,18 +172,24 @@ def fields_update_alias(cls) -> Iterable[str]:
         Maps aliased fields to canonical fields using fields_alias if
         provided.
         """
+
+        if cls.fields_update is None:

```

Comment:
Prefer `assert cls.fields_update is not None` to avoid further unexpected behavior if this hits